### ISSUE TITLE
Game crashing on startup

### ISSUE THREAD

**Issue by MiranaAndTheDiamonds** [ValveSoftware/Dota2-Gameplay#15271]
### Description

My dota keeps crashing everytime I start the game, here is the error message I get is attached

tried every fix I could find, verifying game files, uninstalling and reinstalling the game, moving the game folder, setting allocated memory, starting with -safe and -autoconfig, starting compability mode, same error every time!

### Screenshots

![Screenshot_1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/159504017/90baafd8-67fd-4c7f-a0a2-06c2446a167e)

**Comment by sadistgila** [comment]
im having the same problem, can anyone help?

**Comment by Krhandota2** [comment]
Dude u solve thiss?

**Comment by MiranaAndTheDiamonds** [comment]
No, still waiting for Valve to fix this trash update that fucked up Dota to a point where so many people cant play

**Comment by Krhandota2** [comment]
Dude now i am reinstalling Windows if solve this i can say u bro

**Comment by MiranaAndTheDiamonds** [comment]
I read that reinstalling Windows would solve it, but I am definitely not going to do that because valve can't release proper patches, please valve fix your own shit

**Comment by sadistgila** [comment]
i've done reinstalling windows and reinstall dota the problem still persist, any help?

**Comment by sadistgila** [comment]
dude try this 

try uninstalling dota2, deleting dota 2 beta folder and then installing dota on another hard disk (or create new partition of your hard disk and install it there).
